wrapper/wrapper.go

```diff
@@ -20,13 +20,19 @@ limitations under the License.
 package wrapper
 
 import (
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
 	"github.com/opencontainers/runc/libcontainer/cgroups"
 	"github.com/opencontainers/runc/libcontainer/cgroups/fs"
 )
 
 // Cgroups2Stats holds pointer to appropriate cgroup type (defined in lib `opencontainers`) under cgroup name as a key
 var Cgroups2Stats = map[string]Stats{
```

Comment:
In current implementation we are limited by `openlibcontainers` lib. As far as I know it does not support v2 at this moment.